packages/cli/src/rpc/util.ts

```diff
@@ -22,3 +23,39 @@ export function callWithStackTrace(handler: Function, debug: boolean) {
     }
   }
 }
+
+/**
+ * Converts a BitArray (protocol v0) to a boolean array indicating which indexes are true.
+ * @param res The response from sendOffer as a BitArray
+ * @param contentKeysLength The number of content keys that were offered
+ * @returns Object with success array, declined flag, or failed flag
+ */
+export function bitToBooleanArray(resBitArray: BitArray, contentKeysLength: number): { success: boolean[] } {
+  const acceptedBits = resBitArray.getTrueBitIndexes()
+  const successArray = new Array(contentKeysLength).fill(false)
+  acceptedBits.forEach((index: number) => {
+    if (index < contentKeysLength) {
+      successArray[index] = true
+    }
+  })
+  return { success: successArray }
+}
+
+/**
+ * Converts a Uint8Array (protocol v1) to a boolean array indicating which indexes are true.
+ * @param res The response from sendOffer as a Uint8Array
+ * @param contentKeysLength The number of content keys that were offered
+ * @returns Object with success array, declined flag, or failed flag
+ */
+export function Uint8toBooleanArray(res: Uint8Array, contentKeysLength: number): { success: boolean[] } {
+  const successArray = new Array(contentKeysLength).fill(false)
+  for (let i = 0; i < contentKeysLength; i++) {
+    const byteIndex = Math.floor(i / 8)
+    const bitIndex = i % 8
+    const byte = res[byteIndex]
+    if (byte !== undefined) {
+      successArray[i] = ((byte >> bitIndex) & 1) === 1
+    }
+  }
+  return { success: successArray }
+}
```

Comment:
I think this is giving false results.  With the V1 response, each byte in the Uint8Array represents an `AcceptCode`.  `AcceptCode.ACCEPT = 0`, and all other values represent different types of rejections.  So for each index of the Uint8Array, just check if `res[i] === AcceptCode.ACCEPT`, and that should give you the boolean array.
```suggestion
export function Uint8toBooleanArray(res: Uint8Array, contentKeysLength: number): { success: boolean[] } {
  const successArray = new Array(contentKeysLength).fill(false)
  for (let i = 0; i < contentKeysLength; i++) {
    if (byte !== undefined) {
      successArray[i] = AcceptCode.ACCEPT
    }
  }
  return { success: successArray }
}
```